### ISSUE TITLE
Space bar swipe up is inconsistent

### ISSUE THREAD

**Issue by Elaborendum** [florisboard/florisboard#1546]
#### Short description
Occasionally, swiping up (to change language in my case) the space bar won't do anything. I would try testing it with the beta version, but it doesn't open at all for me.

#### Steps to reproduce
1. Go to Gestures
2. Scroll down to Space bar Gestures and choose "switch to next subtype" as the action for swiping up.
3. Swipe up the space bar continuously (just to make it clearer, but it happens regardless of rhythm), and see that sometimes no action is made.

#### Environment information
- FlorisBoard Version: 0.3.13
- Install Source: F-Droid
- Device: Samsung A20
- Android: 10 stock

**Comment by Albnu14** [comment]
I faced something similar, it was because the keyboard would only accept fast gestures "swipe velocity threshold" 1900 dp/s i changed this to 600dp/s and it solved my problem

Do you think you can try changing this setting from 1900 to another value and see if it solves it?

**Comment by Elaborendum** [comment]
How exactly can I do that? I can't see exact values, only "Very Slow", "Slow", "Normal", "Fast" and "Very Fast". I chose Very Slow, and it seems to have improved, with only very rare misses here and there, although it could just be me tapping the very edge of the key by accident on occasion.

**Comment by Albnu14** [comment]
@Elaborendum 
Yes, that is the setting, it didn't have values in the previous stable, which you use

But In the current beta 9, it provides values, the default is 1900dp/s.

I guess changing the setting to very slow equals the minimum 400dp/s,

If that doesn't solve it, your problem could be something else, maybe the devs can help.

**Comment by lm41** [comment]
This issue has been open for a long period of time. Does it still occur in the latest version?

**Comment by lm41** [comment]
Fixed in: https://github.com/florisboard/florisboard/pull/2959
